### DISCUSSION TITLE
VueJS implementation

### DISCUSSION THREAD

**Discussion by SpiritusDeos** [WalletConnect/walletconnect-monorepo#1948] [Developer Support]
Hi,

What is the best solution to handle `web3modal` implement on a Vue 3 project ?

**Comment by MathieuBurnat** [comment]
Hey @kennyyS , have you found any clues since your comment ?

**Reply by SpiritusDeos** [reply]
I used the native SDK for Javascript, and makes the job

**Reply by Pmurs** [reply]
Do you have any sample code you can show? The official tutorial is pretty sparse

**Comment by chabroA** [comment]
Hey @kennyyS 👋 

Not familiar with Vue3 but maybe this should do the job -> https://github.com/mfw78/web3modal-vue3

**Reply by SpiritusDeos** [reply]
Thanks @chabroA 🙏🏻

**Reply by Pmurs** [reply]
@chabroA That module looks pretty old -- seems like WalletConnect V2 support will be an issue there?

**Reply by chabroA** [reply]
@Pmurs most probably indeed, since WalletConnect V2 is fairly new and in the process of being rolled out, there is probably not a lot of packages comptable with it yet (except [WalletConnect own SDKs](https://docs.walletconnect.com/2.0/) but which might be more tailored to React since leveraging [wagmi](https://github.com/wagmi-dev/wagmi))

**Reply by SpiritusDeos** [reply]
There is @Pmurs web3-onboard which is pretty easier to use than build the solution by yourself 👍🏻
